Makefile

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+PROJECT := gangway
+# Where to push the docker image.
+REGISTRY ?= gcr.io/heptio-images
+IMAGE := $(REGISTRY)/$(IMAGE)
```

Comment:
should this be `IMAGE := $(REGISTRY)/$(PROJECT)` ?

Reply:
Bah, yes, just pushed the fix and rebased.